src/Components/LandingPage/ExerciseListElement.tsx

```diff
@@ -1,4 +1,42 @@
-type Props = { exercise: string };
-export const ExerciseListElement = ({ exercise }: Props) => {
-  return <h1>{exercise}</h1>;
+import { ChartBar, Play } from 'phosphor-react';
+import { useNavigate } from 'react-router-dom';
+import { MouseEvent } from 'react';
+
+const EXERCISE: string = 'exercise';
+type Props = { title: string; description: string };
+
+export const ExerciseListElement = ({ title, description }: Props) => {
+  let navigate = useNavigate();
+  const handleClick = (e: MouseEvent<HTMLElement>) => {
```

Comment:
```suggestion
  let navigate = useNavigate();
  
  const handleClick = (e: MouseEvent<HTMLElement>) => {
```
Det er almindeligt at have et whitespace imellem ens hooks og andre funktioner i ens component